src/lib/layout/Sidebar.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import NavItem from '$lib/navbar/NavItem.svelte';
+  import BackgroundHexagon from '$lib/layout/BackgroundHexagon.svelte';
+  import Icon from '$lib/component/Icon.svelte';
+  import Icons from '$lib/component/Icons';
+
+  let selectedIndex: number | null = null;
+
+  const items = ['Início', 'Eventos', 'Projetos', 'Equipa', 'Contactos'];
+
+  let sidebarClosed = true;
+</script>
+
+{#if sidebarClosed}
+  <nav
+    class="bg-transperent fixed grid h-fit w-full grid-cols-[1fr_4em] grid-rows-1 justify-items-center py-4 px-2 text-white"
+  >
+    <button
+      class="col-start-2 h-fit w-1/2 sm:invisible"
+      on:click={() => (sidebarClosed = !sidebarClosed)}
+    >
+      <Icon src={Icons.Bars} color="#411315" size="31px" href={undefined} />
```

Comment:
Do we want this color to be hard-coded (genuine question)?

Reply:
I believe that, for now, we can leave this color hard-coded seeing as it is being defined in svelte rather than in tailwind and those color definitions are not accessible from within svelte (as far as I'm aware). However if in the future this proves to be a recurring issue we can create a separate file to store color declarations to be used in svelte.